cucumber/authenticators_jwt/features/authn_jwt_fetch_identity_decoded_token.feature

```diff
@@ -0,0 +1,191 @@
+        resource: !webservice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+
+    - !host
+      id: some_policy/sub_policy/host_test_from_token
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host some_policy/sub_policy/host_test_from_token
+
+    - !host
+      id: some_policy/host_test_from_token
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host some_policy/host_test_from_token
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+
+  Scenario: A valid JWT token with identity in the token
+    Given I have a "variable" resource called "test-variable"
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I permit host "myapp" to "execute" it
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then host "myapp" has been authorized by Conjur
+    And I successfully GET "/secrets/cucumber/variable/test-variable" with authorized user
+    And The following appears in the log after my savepoint:
+    """
+    cucumber:host:myapp successfully authenticated with authenticator authn-jwt service cucumber:webservice:conjur/authn-jwt/raw
+    """
+
+  Scenario: User as Token identity is not supported, error
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    Given I extend the policy with:
+    """
+    - !user
+      id: myuser
+      annotations:
+        authn-jwt/raw/custom-claim:
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !user myuser
+    """
+    And I issue a JWT token:
+    """
+    {
+      "host":"myapp2",
+      "project-id": "myproject"
+    }
+    """
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+
+  Scenario: Token identity configuration no matching any claim, error
+    Given I issue a JWT token:
+    """
+    {
+      "project-id": "myproject"
+    }
+    """
+    And I successfully set authn-jwt "token-app-property" variable to value "host_cliam"
+    And I save my place in the log file
+    When I authenticate via authn-jwt with the JWT token
+    Then the HTTP response status code is 401
+    And The following appears in the log after my savepoint:
+    """
+    Errors::Authentication::AuthnJwt::NoSuchFieldInToken: CONJ00081E
```

Comment:
We already agreed that you dont need to search for `Errors::Authentication::AuthnJwt::NoSuchFieldInToken`, you need to search for the `CONJ00081E' and the error message (which you didnt search for at all)

Reply:
Done all over the authn_jwt tests